frontend/public/components/modals/disable-application-modal.tsx

```diff
@@ -24,14 +24,21 @@ export class DisableApplicationModal extends PromiseComponent {
     const deleteOptions = {kind: 'DeleteOptions', apiVersion: 'v1', propagationPolicy: 'Foreground'};
     const killPromises = [k8sKill(SubscriptionModel, subscription, {}, deleteOptions)]
       .concat(_.get(this.props.subscription, 'status.installedCSV') && this.state.deleteCSV
-        ? k8sKill(ClusterServiceVersionModel, {metadata: {name: subscription.status.installedCSV, namespace: subscription.metadata.namespace}} as ClusterServiceVersionKind, {}, deleteOptions)
+        ? k8sKill(ClusterServiceVersionModel, {metadata: {name: subscription.status.installedCSV, namespace: subscription.metadata.namespace}} as ClusterServiceVersionKind, {}, deleteOptions).catch(() => Promise.resolve())
```

Comment:
Do we not want to report errors?

Reply:
We are assuming the CSV exists by a field on the `Subscription`; if we try and delete it but it doesn't exist, then that's fine.

Reply:
Do we need to check if it failed for a reason other than 404?

I'd think this at least deserves a comment explaining

Reply:
I'm OK with this behavior, but I think we want a comment in the code